.github/workflows/build-and-test.yml

```diff
@@ -10,6 +10,51 @@ on:
     branches:
       - main
 jobs:
+  TestsWithArm:
```

Comment:
Nice work getting this working at all 👍 

Does this run in parallel with the rest of the test matrix? 

Those timings are grim 13 min to build and 22 to test. Are we targeting a single framework when we build?

Reply:
> Does this run in parallel with the rest of the test matrix?

Yes it does

> Those timings are grim 13 min to build and 22 to test. Are we targeting a single framework when we build?

Only net 5.0